content/blog/streamline-and-optimize-ml-workflows-with-hpe-ezmeral-unified-analytics.md

```diff
@@ -0,0 +1,153 @@
+
+Next, a new model server must be created in Kube Flow and the yaml file uploaded. The inference service is created and once it is running, we can send our test data through REST APIs and get the response.
+
+**Model Retraining**
+
+The models deployed into production are constantly monitored to identify inconsistencies and degradation in performance. Because models can skew to the old data used for training resulting in under performance, models should be retrained whenever performance drops below a defined threshold. To retrain a model, a pipeline is developed that could trigger all the steps associated with model retraining to occur in the appropriate sequence using Kube Flow pipelines.  
+
+Python programs need to be modified to run as a Kube Flow pipeline because each step runs as a containerized module. The whole cycle of data collection, preprocessing, and modeling are divided into separate components with each process isolated and running as independent containers inside individual Kubernetes pods. All the dependencies of each task must be mentioned inside the task due to the isolated nature of the environment. The artifacts that are used in the process, such as input data files and model objects, are saved in specific locations and referenced by the tasks. 
+
+To prepare the Kube Flow pipelines, we have to define each task as a function. First “read_data()” is defined to read the input file from the data volume on the Kube Flow pipeline.
+
+![](/img/fig-1-define-each-task.png "Figure 10. Each task is defined as a function that includes all necessary packages. ")
+
+Once defined, functions are converted into a Kube Flow component using “create_component_from_func()”.
+
+![](/img/fig-2-converting-each-task.png "Figure 11. Each task is converted into a component that includes base line images and packages. ")
+
+Once each of the tasks are defined, the Kube Flow pipelines are defined to call each of the steps using DSL (Domain Specific Language).
+
+![](/img/fig3-define-the-pipeline.png "Figure 12. Pipelines are defined along with tasks in the appropriate sequence. ")
+
+When the Kube Flow pipeline is set up, the pipeline can run using the Kube Flow client where the required arguments and parameters are passed for the specific run.
+
+![](/img/kubeflow-setup-run.png "Figure 13. Kube Flow client is created and executed with appropriate arguments. ")
+
+In the Kube Flow UI, “stock_pipeline_run” would be triggered to provide a green-red status of each step as it executes. 
+
+![](/img/fig-5-simplified-pipeline.png "Figure 14. Status of each pipeline step and it's status. ")
+
+**Conclusion**
+
+From the above example, HPE Ezmeral Unified Analytics has significantly enhanced the end-to-end pipeline of financial service use cases by providing a comprehensive platform tailored to the specific needs of ML practitioners and data scientists. Here is a conclusion highlighting the benefits of HPE Ezmeral Unified Analytics:
```

Comment:
I'd recommend combining these two paragraphs to read something like this:

As you can see, HPE Ezmeral Unified Analytics has significantly enhanced the end-to-end pipeline of the financial service use cases by providing a comprehensive platform tailored to the specific needs of ML practitioners and data scientists. HPE Ezmeral Unified Analytics offers seamless integration of various data engineering and ML components, like Spark, EzPresto, Superset, MLflow, and Kube Flow. This integration streamlines the entire ML workflow, reducing friction between dissimilar stages and enables more efficient development of ML pipelines. 

In summary: (and then it goes to your bullets)

(or something like that at the end... feels like it's missing a few words at the very end. Also, you didn't need "Here is a conclusion highlighting the benefits of..."